JSDemos/Demos/DataGrid/RecordPaging/AngularJS/data.js

```diff
@@ -1,201 +1,33 @@
-    "State" : "California",
-    "Zipcode" : 90034,
-    "Phone" : "(310) 237-5642",
-    "Fax" : "(310) 237-5643",
-    "Website" : "http =//www.nowebsitezonetoys.com"
-}, {
-    "ID" : 12,
-    "CompanyName" : "ACME",
-    "Address" : "2525 E El Segundo Blvd",
-    "City" : "El Segundo",
-    "State" : "California",
-    "Zipcode" : 90245,
-    "Phone" : "(310) 536-0611",
-    "Fax" : "(310) 536-0612",
-    "Website" : "http =//www.nowebsiteacme.com"
-}, {
-    "ID" : 13,
-    "CompanyName" : "Super Mart of the West",
-    "Address" : "702 SW 8th Street",
-    "City" : "Bentonville",
-    "State" : "Arkansas",
-    "Zipcode" : 72716,
-    "Phone" : "(800) 555-2797",
-    "Fax" : "(800) 555-2171",
-    "Website" : "http://www.nowebsitesupermart.com"
-}, {
-    "ID" : 14,
-    "CompanyName" : "Electronics Depot",
-    "Address" : "2455 Paces Ferry Road NW",
-    "City" : "Atlanta",
-    "State" : "Georgia",
-    "Zipcode" : 30339,
-    "Phone" : "(800) 595-3232",
-    "Fax" : "(800) 595-3231",
-    "Website" : "http =//www.nowebsitedepot.com"
-}, {
-    "ID" : 15,
-    "CompanyName" : "K&S Music",
-    "Address" : "1000 Nicllet Mall",
-    "City" : "Minneapolis",
-    "State" : "Minnesota",
-    "Zipcode" : 55403,
-    "Phone" : "(612) 304-6073",
-    "Fax" : "(612) 304-6074",
-    "Website" : "http =//www.nowebsitemusic.com"
-}, {
-    "ID" : 16,
-    "CompanyName" : "Tom's Club",
-    "Address" : "999 Lake Drive",
-    "City" : "Issaquah",
-    "State" : "Washington",
-    "Zipcode" : 98027,
-    "Phone" : "(800) 955-2292",
-    "Fax" : "(800) 955-2293",
-    "Website" : "http =//www.nowebsitetomsclub.com"
-}, {
-    "ID" : 17,
-    "CompanyName" : "E-Mart",
-    "Address" : "3333 Beverly Rd",
-    "City" : "Hoffman Estates",
-    "State" : "Illinois",
-    "Zipcode" : 60179,
-    "Phone" : "(847) 286-2500",
-    "Fax" : "(847) 286-2501",
-    "Website" : "http =//www.nowebsiteemart.com"
-}, {
-    "ID" : 18,
-    "CompanyName" : "Walters",
-    "Address" : "200 Wilmot Rd",
-    "City" : "Deerfield",
-    "State" : "Illinois",
-    "Zipcode" : 60015,
-    "Phone" : "(847) 940-2500",
-    "Fax" : "(847) 940-2501",
-    "Website" : "http =//www.nowebsitewalters.com"
-}, {
-    "ID" : 19,
-    "CompanyName" : "StereoShack",
-    "Address" : "400 Commerce S",
-    "City" : "Fort Worth",
-    "State" : "Texas",
-    "Zipcode" : 76102,
-    "Phone" : "(817) 820-0741",
-    "Fax" : "(817) 820-0742",
-    "Website" : "http =//www.nowebsiteshack.com"
-}, {
-    "ID" : 20,
-    "CompanyName" : "Circuit Town",
-    "Address" : "2200 Kensington Court",
-    "City" : "Oak Brook",
-    "State" : "Illinois",
-    "Zipcode" : 60523,
-    "Phone" : "(800) 955-2929",
-    "Fax" : "(800) 955-9392",
-    "Website" : "http =//www.nowebsitecircuittown.com"
-}];
\ No newline at end of file
+let s = 123456789;
+function random() {
+    s = (1103515245 * s + 12345) % 2147483647;
+    return s % (10 - 1);
+}
+
+const generateData = function (count) {
+    var i;
```

Comment:
Early, we use a single `var` for a block of variables declaration. Maybe to make it similar.